UI/obs-app.cpp

```diff
@@ -2122,6 +2122,22 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 		     rosettaTranslated ? "true" : "false");
 #endif
 
+#if defined(_WIN32)
+		if (IsEmulatedOnARM64Windows()) {
+#if defined(_WIN64)
+			blog(LOG_INFO,
+			     "Windows ARM64: Running with X64 Emulation");
+#else
+			blog(LOG_INFO,
+			     "Windows ARM64: Running with X86 Emulation");
```

Comment:
I don't believe ARM64 has x86 emulation, and by the time we have a Windows ARM build, we won't support x86 OBS anyway (it exists mostly for legacy reasons, to support 32-bit VSTs and things). I'd be OK with ignoring that use case. Otherwise I like this wording much better.

Reply:
Believe or not, when Surface Pro X launched, Windows ARM64 only comes with x86 emulation. Until sometime last year in Windows Insider Dev branch, Windows on ARM64 gained X64 emulation. X64 emulation should come with now dead Windows 10 Sun Vally, later the Sun Vally become the Windows 11, and X64 emulation is a Windows 11 exclusive feature now.

And you are right, we don't need x86 anymore if there's X64.